eng/testing/RunnerTemplate.sh

```diff
@@ -132,7 +132,9 @@ if [[ "$(uname -s)" == "Darwin" ]]; then
   # files already in /cores/ at this point. This is being done to prevent
   # inadvertently flooding the CI machines with dumps.
   if [[ ! -d "/cores" || ! "$(ls -A /cores)" ]]; then
-    ulimit -c unlimited
+    # FIXME: temporarily disable core dumps
```

Comment:
If we merge this, please open an issue to track re-enabling it. Thanks!

Reply:
yeah will do, just wanted to validate first that this actually works

Reply:
Ok, seems to work for the System.Runtime.InteropServices.Tests issue, but the PayloadGroup0 runtime test is also still running into it because it also sets `ulimit`.